src/main/java/org/broadinstitute/hellbender/engine/AlignmentContext.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Bundles together a pileup and a location.
  */
-public final class AlignmentContext implements HasGenomeLocation {
+public final class AlignmentContext implements Locatable, HasGenomeLocation {
```

Comment:
Could you rebase onto the latest master? `AlignmentContext` has already been patched to implement `Locatable`